src/enums/routeNames.ts

```diff
@@ -4,7 +4,10 @@ export enum RouteNames {
   CHANGE = 'change',
   CONVERSION = 'conversion',
   CORRECTION = 'correction',
-  RESTORATION = 'restoration',
+  RESTORATION_FULL = 'fullRestoration',
+  RESTORATION_LIMITED = 'limitedRestoration',
+  RESTORATION_EXTENSION = 'limitedRestorationExtension',
+  RESTORATION_CONVERSION = 'limitedRestorationToFull',
```

Comment:
Just something super small, do you think it would be better to name this `'limitedRestorationConversion'`? This way we would be consistent with the naming (sorry for being nitpicky).

Reply:
I noticed this too but it would require a small change to the scheme and API.

@argush3 @vysakh-menon @kzdev420 What do you think? Is that something you small we should do, something Kevin or Karim could do?

Reply:
Oh wait -- I was thinking this was the filing subtype field (not the route). I have no issue with whether you want to keep or change this.

Reply:
I was thinking the same. Extension and conversion are sub-types. Wondering if this changes anything in terms of initial load of the draft.

Reply:
Old world describes the conversion filing as "Restoration Application (Convert Limited to Full)".  We can update it and I do think it's something Karim or Kevin can work on.

As for whether we should do it, I don't think it's required unless people really feel it's confusing.

Reply:
If we're going to change the subtypes then now is the second-cheapest time to do it. Any strong opinions on whether we should do it?